qe_metrics/manifests/creds.example.yaml

```diff
@@ -1,10 +1,12 @@
 database:
     host: <FQDN or IP of the database server>
-    port: <Database service port number>
+    port: <Database service port number. Default 5432.>
```

Comment:
should be in readme

Reply:
@rnetser I have removed from this file and will open a new PR to add credential configuration to the readme

Reply:
Done in #39